skein/utils.py

```diff
@@ -82,12 +84,15 @@ def grpc_fork_support_disabled():
     gRPC (>= 1.15) results in extraneous error logs currently. For now we
     explicitly disable fork support for gRPC clients we create.
     """
-    key = 'GRPC_ENABLE_FORK_SUPPORT'
-    try:
-        os.environ[key] = '0'
+    if LooseVersion(pkg_resources.get_distribution("grpcio").version) < '1.18.0':
```

Comment:
`pkg_resources` is also not standard library. Instead it looks like `grpc` version may be found at:

```python
try:
    from grpc import __version__ as GRPC_VERSION
except AttributeError:
    from grpc._grpcio_metadata import __version__ as GRPC_VERSION
```

from https://github.com/grpc/grpc/pull/17580.